android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java

```diff
@@ -0,0 +1,165 @@
+package com.liferay.mobile.screens.viewsets.defaultviews.ddl.form.fields;
+
+import android.content.Context;
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.support.annotation.RequiresApi;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import com.liferay.mobile.screens.R;
+import com.liferay.mobile.screens.ddl.form.view.DDLFieldViewModel;
+import com.liferay.mobile.screens.ddl.model.GeoLocation;
+import com.liferay.mobile.screens.ddl.model.GeolocationField;
+import com.liferay.mobile.screens.util.LiferayLocale;
+import java.text.NumberFormat;
+import java.text.ParseException;
+
+/**
+ * @author Víctor Galán Grande
+ */
+
+public class DDLFieldGeoView extends LinearLayout implements DDLFieldViewModel<GeolocationField>, TextWatcher {
+
+
+	public static final double INVALID_LATITUDE = -91;
```

Comment:
?

Reply:
I inlined the value, but it was very "magic"

Reply:
Why not -90 or -180/2 and +1/-1